internal/helm/chart/builder.go

```diff
@@ -104,6 +104,10 @@ type BuildOptions struct {
 	// Force can be set to force the build of the chart, for example
 	// because the list of ValuesFiles has changed.
 	Force bool
+
+	// Keyring can be set to the data of the chart VerificationKeyring secret
+	// used for verifying a chart's signature using a provenance file.
```

Comment:
It is better to document directly what the `[]byte` is expected to contain, than to refer to elsewhere. For example:

```suggestion
	// Keyring can be configured with the bytes of a public keyring in legacy GPG format,
	// used for verifying a chart's signature using a provenance file.
```